testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -551,18 +564,17 @@ def test_sequence_Seq(self):
         s = p.residues.sequence(format="Seq")
         assert_equal(s.tostring(), self.ref_adk_sequence)
 
-    # INVALID: should set resnames with `AG.residues.resnames = 'MSE'`
-    @skip
     def test_sequence_nonIUPACresname(self):
         """test_sequence_nonIUPACresname: non recognized amino acids raise
-        ValueError"""
```

Comment:
This should stay as ValueError, it's probably AE atm because we've not got sequence in place yet